src/idl_gen_swift.cpp

```diff
@@ -139,6 +139,7 @@ class SwiftGenerator : public BaseGenerator {
   bool generate() {
     code_.Clear();
     code_.SetValue("ACCESS", "_accessor");
+    code_.SetValue("Spacer", "\t\t");
```

Comment:
why are we using tabs for indentation? I couldn't get a conclusive answer, but it seems 4 spaces is most common for Swift? We output spaces for all languages except Go, so unless Swift mandates tabs, we should use spaces here too. And its more readable to put them straight in the generated code instead of using `{{SPACER}}` all over the place.

Reply:
I've decided to go with code_.IncrementIndentation() for spacing, we only set the spacing before hand in the code writer and some places that we really need to specify the indentation amount.